Resources/doc/installation.rst

```diff
@@ -78,6 +78,20 @@ the MongoDB ODM across your application:
     Of course, you'll also need to make sure that the MongoDB server is running
     in the background. For more details, see the MongoDB `Quick Start`_ guide.
 
+
+Authentication
+--------------
+
+If you use authentication on your MongoDB database you can provide username, password, and authentication database in the following way:
```

Comment:
Please comply with the sf docs standards (I think we use something similar for Doctrine):

> [Each line should break approximately after the first word that crosses the 72nd character (so most lines end up being 72-78 characters);](http://symfony.com/doc/current/contributing/documentation/standards.html)